ert3/evaluator/_builder.py

```diff
@@ -84,7 +77,7 @@ async def transform_output(
             .set_name(command.name)
             .set_transformation(
                 ert.data.ExecutableTransformation(
-                    location=command_location(command.name)
+                    location=pathlib.Path(stage.command_location(command.name).name)
```

Comment:
Equivalent to the newly created function? 
```suggestion
                    location=stage.command_final_path_component(name)
```